styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,74 @@
 .footer{
-  
-}
\ No newline at end of file
+  .accordionTitle h6{
+    color: $white;
+    font-size: 1.3rem;
+  }
+  ul{
+    list-style: none;
+    @extend .no-padding;
+    @extend .no-margin;
+    li{
+      @extend .no-padding;
+      margin:5px 0px;
+      a{
+        color: darken($light-gray, 15%);
+        font-size: 1.2rem;
+      }
+    }
+    &.split{
+      column-count: 3;
+    }
+  }
+}
+/*Mobile*/
+@media only screen and (min-width:768px) and (max-width: 991px){
+  .full-width-small{
+    width:100%;
+  }
+}
+@media only screen and (min-width:50px) and (max-width: 992px){
+  .fluid-footer-container{
+    padding:0px 0px;
+  }
+  .footer{
+    padding:0px 0px;
+    .accordionTitle{
+      background-color: $blue-gray;
+      border-bottom:1px solid lighten($blue-gray, 05%);
+      h6{
+        text-align: center;
+        padding:12px;
+        margin: 0px 0px;
+      }
+      &:hover{
+        background-color: darken( $blue-gray, 05%);
+        cursor:pointer;
+        cursor: hand;
+      }
+    }
+    .accordionMenu{
```

Comment:
Similar to the comment above on `.accordionTitle` this could be a standalone class `.footer-accordion-menu` that is more efficient and modular.